src/App.tsx

```diff
@@ -1,25 +1,15 @@
-/**
- * NOTE: You will need to `npm link` zUI before this repo
- * will build or run.
- */
-
-//- React Imports
 import { FC } from 'react';
 
-//- Types Imports
 import { AppProps } from './lib/types/app';
 
-//- Hook Imports
 import { useDataContainer } from './lib/hooks/useDataContainer';
 
-//- Container Imports
-import Domains from './pages/Domains/Domains';
-import NFT from './pages/NFT/NFT';
+import { Domains } from './pages/Domains/Domains';
+import { NFT } from './pages/NFT/NFT';
```

Comment:
nit: if we have `pages/Domains/index.ts` with `export * from './Domains.tsx`. then we can use `import { Domains } from './pages/Domains';`

Reply:
👍